rules/CodingStyle/Application/UseImportsAdder.php

```diff
@@ -159,26 +177,35 @@ private function resolveInsertNop(FileWithoutNamespace|Namespace_ $namespace): a
 
     /**
      * @param Stmt[] $stmts
-     * @param Use_[] $newUses
+     * @return Comment[]
      */
-    private function mirrorUseComments(array $stmts, array $newUses, int $indexStmt = 0): void
+    private function getAndRemoveExistingUseComments(array $stmts, int $indexStmt = 0): array
```

Comment:
We split this function in two, the first part removes the comments and returns them, the second part reapplies them